siem_integrations/README.md

```diff
@@ -26,10 +26,103 @@ In order to utilize splunk2kafka, a [running Kafka instance](https://kafka.apach
 2. Install export2kafka ([Instructions](https://github.com/rapidsai/clx-siem-integration/blob/master/splunk2kafka/export2kafka/README.md))
 
 
```

Comment:
There should be a more detailed explanation about what CLX Query is here, before we dive directly into downloading the sample dataset. Give a brief (1-2 sentences) overview of the purpose.

Also, there should be a Requirements section that lists all of the requirements (including minimum versions), complete with web links to the appropriate pages for Django, Gunicorn, etc.

Reply:
It would be good to reorder this whole section as the following:

- **CLX Query**
  - Overview
  - Requirements
  - Download Sample Dataset
  - Install and Run CLX Query Service
  - Install and Run CLX Query
  - Known Issues

In the overview section, provide a brief description of all the components and how they link together. It also might be good to have a diagram (you made one already).

Reply:
Reordered `CLX Query` section as suggested. Thank you!